tank_lab_to_nwb/convert_towers_task/towersnwbconverter.py

```diff
@@ -74,4 +74,12 @@ def get_metadata(self):
             TowersPosition=dict()
         )
 
+        if os.path.isfile(session_path + ".mat"):
+            session_data = convert_mat_file_to_dict(mat_file_name=session_path)
+            subject_data = session_data['log']['animal']
+            key_map = dict(name='subject_id')
+            [metadata['Subject'].update({key_map[k]: str(subject_data[k])}) for k in key_map if k in subject_data]
```

Comment:
If we only use `name` from `subject_data`, why need a loop?
Can we use just a simpler form?

```
if 'name' in subject_data:
    metadata['Subject'].update({'subject_id': str(subject_data['name'])})
```

Reply:
I wouldn't worry about it at this point; it's on the ToDo list to look into how difficult it would be to automatically pull it from the subject database anyway.

Reply:
@weiglszonja good spot. If you press the icon with "+ -" that will allow you to suggest changes that can be committed directly into the PR